exercises/1-the-manual-menace/README.md

```diff
@@ -212,30 +241,36 @@ cd enablement-ci-cd
 ansible-galaxy install -r requirements.yml --roles-path=roles
 ```
 
-11.   Apply the inventory by logging into OpenShift on the terminal. You will need to retrieve a token first, by browsing to the token request page. This is also available from the `Copy Login Command` once you have logged into the OpenShift Web UI. (`<CLUSTER_URL>` should be replaced with the one you've been provided by the instructor). Accept any insecure connection warning(s) from the cli 👍:
+11. Apply the inventory by logging into OpenShift on the terminal. You will need to retrieve a token first, by browsing to the token request page. This is also available from the `Copy Login Command` once you have logged into the OpenShift Web UI. (`<CLUSTER_URL>` should be replaced with the one you've been provided by the instructor). Accept any insecure connection warning(s) from the cli 👍:
```

Comment:
In FAQ section I added a picture showing where the 'copy login command' is in the console ui if we want to add that to make this more clear